clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/api/v1/model/CheckResult.groovy

```diff
@@ -19,29 +19,29 @@ package com.netflix.spinnaker.clouddriver.consul.api.v1.model
 import com.fasterxml.jackson.annotation.JsonProperty
 
 class CheckResult {
-  @JsonProperty("Node")
-  String node
+  @JsonProperty("node")
```

Comment:
Can you use http://fasterxml.github.io/jackson-annotations/javadoc/2.0.0/com/fasterxml/jackson/annotation/JsonSetter.html instead of capitalizing the field names?